src/config/constants.ts

```diff
@@ -232,8 +232,173 @@ export enum SERVER_MODE {
 export type SERVER_MODE_TYPE = keyof typeof SERVER_MODE;
 
 export enum ACCESS_TYPE_MAP {
-  DEVTRON_APPS = '', // devtron app work flow
-  HELM_APPS = 'helm-app', //helm app work flow
+    DEVTRON_APPS = '', // devtron app work flow
+    HELM_APPS = 'helm-app', //helm app work flow
 }
 
 export const HELM_APP_UNASSIGNED_PROJECT = 'unassigned';
+
+export const REGISTRY_TYPE_MAP = {
```

Comment:
Create a specific typed map or record (if needed to be kept as an object).
**E.g.** const REGISTRY_TYPE_MAP: Record<string, RegisteryType> = {ecr: {...}}